src/jquery.multiselect.js

```diff
@@ -83,27 +77,26 @@
-      linkInfo: null,                     // (plain object | null) Supply an obect of link information to use alternative icons, icon labels, or icon title text.  See linkDefaults above for object structure.
-      noneSelectedText: 'Select options', // (str | null) The text to show in the button where nothing is selected.  Set to null to use the native select's placeholder text.
-      selectedText: '# of # selected',    // (str) A "template" that indicates how to show the count of selections in the button.  The "#'s" are replaced by the selection count & option count.
-      selectedList: 0,                    // (int) The actual list selections will be shown in the button when the count of selections is <= than this number.
-      selectedListSeparator: ', ',        // (str) This allows customization of the list separator.  Use ',<br/>' to make the button grow vertically showing 1 selection per line.
-      maxSelected: null,                  // (int | null)  If selected count > maxSelected, then message is displayed, and new selection is undone.
-      show: null,                         // (array) An array containing menu opening effects.
-      hide: null,                         // (array) An array containing menu closing effects.
+      zIndex: null,                       // (integer) Overrides the z-index set for the menu container.
+      classes: '',                        // (string) Classes that you can provide to be applied to the elements making up the widget.
+      header: ['checkAll','uncheckAll'],  // (false | string | array) False, custom string or array indicating which links to show in the header & in what order.
+      linkInfo: null,                     // (object | null) Supply an obect of link information to use alternative icons, icon labels, or icon title text.  See linkDefaults above for object structure.
+      noneSelectedText: 'Select options', // (string | null) The text to show in the button where nothing is selected.  Set to null to use the native select's placeholder text.
+      selectedText: '# of # selected',    // (string) A "template" that indicates how to show the count of selections in the button.  The "#'s" are replaced by the selection count & option count.
+      selectedList: 0,                    // (integer) The actual list selections will be shown in the button when the count of selections is <= than this number.
+      selectedListSeparator: ', ',        // (string) This allows customization of the list separator.  Use ',<br/>' to make the button grow vertically showing 1 selection per line.
+      maxSelected: null,                  // (integer | null)  If selected count > maxSelected, then message is displayed, and new selection is undone.
+      openEffect: null,                   // (array) An array containing menu opening effect information.
+      closeEffect: null,                  // (array) An array containing menu closing effect information.
```

Comment:
I changed these option names to be more clear about what they actually do.  However, I also tend to think that the widget may be relying on deprecated jQuery UI API signatures for these, as current docs mention the use of objects for animations.   Therefore, these need to be re-looked at to see if they should be updated to reflect the current API.  I will post an issue for this.